sb_vision/calibration/cli.py

```diff
@@ -0,0 +1,68 @@
+"""Command-line tool for camera calibration."""
+
+import sys
+import lzma
+import pickle
+import argparse
+from pathlib import Path
+
+import yaml
+
+from .fit import fit
+from .training_example import TrainingExample
+
+
+def argument_parser():
+    """Calibration tool argument parser."""
+    parser = argparse.ArgumentParser(
+        description="Compute calibration from camera",
+    )
+
+    parser.add_argument(
+        'directory',
+        type=Path,
+        help="calibration directory",
+    )
+
+    parser.add_argument(
+        '-o',
+        '--output',
+        type=Path,
+        help="file to save into",
+    )
+
+    return parser
+
+
+def main(args=sys.argv[1:]):
```

Comment:
This feels wrong even though there most likely won't be any problems due to it only be called once.

```python
def main(args=None):
    if args is not None:
        args = sys.argv[1:]
```